crest/Assets/Crest/Crest/Scripts/Shapes/OceanWaveSpectrum.cs

```diff
@@ -16,13 +16,20 @@ namespace Crest
     [CreateAssetMenu(fileName = "OceanWaves", menuName = "Crest/Ocean Wave Spectrum", order = 10000)]
     public class OceanWaveSpectrum : ScriptableObject
     {
+        /// <summary>
+        /// The version of this asset. Can be used to migrate across versions. This value should
+        /// only be changed when the editor upgrades the version.
+        /// </summary>
+        [SerializeField, HideInInspector]
+        int _version = 0;
```

Comment:
Just bringing over my reply in private repository about setting the version to 1:
> You could use Reset to set the version to 1? Since that will only be called on new instances or when the use hits reset button which should then auto upgrade anyway?

Reply:
hmm, need to think more about it

Reply:
this was the missing piece. good shout!